InsertData/src/InsertData.java

```diff
@@ -0,0 +1,309 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.Document;
+import org.json.JSONException;
+import org.json.simple.JSONArray;
+//import org.json.JSONObject;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.util.Arrays;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.util.JSONParseException;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class InsertData {
+		/*  public static void main(String[] args)
+	{
+	System.out.println("hello");
+		String CONNECTION = "mongodb+srv://m001-student:student123#@sandbox-trhqa.mongodb.net/test";
+		//List<Document> check = AddOneOrder(CONNECTION);//, String[] cities)
+		//List<Document> checkAll = AddMultipleOrders(CONNECTION);//, String[] cities)
+	//	AddOneOrderJsonFile
+		//System.out.println(check);
+		//System.out.println(checkAll);
+	
+	List<Document> checkFile = AddMultipleOrderWithData(CONNECTION, sku,item, unit,quant);
+	List<Document> checkFile2 = AddOneOrderWithData(CONNECTION, "a","b", 5.5,5);
+	
+	}
+	*/
+	
+	private static final boolean FileNotFoundException = false;
+
+	/**
+	 * Inserts documents into the collection using insertOne()
+	 * @param connectionString: To MongoDB instance/MongoDB Cluster.
+	 * @param sku: SKU number of the item ordered, item: name of the ide, unit_price: unit price of the item, quantity: number of items ordered
+	 * @return List<Document>: ArrayList of matching documents.
+	 */
+	public static List<Document> AddOneOrderWithData(String connectionString, String sku, String item, Double  unit_price, int quantity)
```

Comment:
1. You could rename parameter 'sku' to stockKeepingUnit to improve readability.
2. You could rename 'unit_price' to unitPrice so that same naming convention is followed everywhere.

Reply:
Changed unit_price and sku as suggested